### ISSUE TITLE
Ability to use FIT files that contains workout

### ISSUE THREAD

**Issue by devdems** [GoldenCheetah/GoldenCheetah#3701]
Looks like GC can't read FIT files containing workout.

I've added an example here. Is there anything that could be done to use those the same as erg file?

**Comment by amtriathlon** [comment]
There is a pending request to generate workout FIT files in Train (https://github.com/GoldenCheetah/GoldenCheetah/issues/2352), currently they are not supported in any way.

**Comment by devdems** [comment]
Hi,

i though https://github.com/GoldenCheetah/GoldenCheetah/issues/2352 is for exporting FIT files?

I'm asking here for importing it.

**Comment by amtriathlon** [comment]
They are not supported in any way, means they cannot be used in Train for import nor export, see https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ-TRAIN#which-workout-formats-are-supported-to-controls-smart-trainers

**Comment by devdems** [comment]
Yeah, i understand that it's currently not supported. But you pointed to the different request for exporting.

So you are saying that this request https://github.com/GoldenCheetah/GoldenCheetah/issues/2352 will cover also the import of those files and not just export? If that is the case then this request should be closed.

**Comment by amtriathlon** [comment]
No, I pointed to that request because it is related and the format is documented there.
Do you plan to work on this feature?

**Comment by TheRealFlyingDutchman** [comment]
I plan to work on this feature. But first I have some questions about coding style and so on. Who can I write a message to regarding the questions?

**Comment by amtriathlon** [comment]
> I plan to work on this feature. But first I have some questions about coding style and so on. Who can I write a message to regarding the questions?

We have some guidelines in https://github.com/GoldenCheetah/GoldenCheetah/wiki/Guidelines-for-submitting-a-patch and
https://groups.google.com/forum/#!forum/golden-cheetah-developers to ask questions or get help.
Thanks.
